### PR TITLE
Problem: Ambiguous message while importing an existing transaction (f…

### PR DESCRIPTION
**Pull request by leejw51crypto**
If the tx is already imported, showed message "This transaction is already imported/in your wallet"

### CODE REVIEW

File: client-core/src/wallet/default_wallet_client.rs [hunk 1]
```diff
@@ -1066,6 +1066,15 @@ where
     /// import a plain base64 encoded plain transaction
     fn import_plain_tx(&self, name: &str, enckey: &SecKey, tx_str: &str) -> Result<Coin> {
         let tx_info = TransactionInfo::decode(tx_str)?;
+
+        let found_tx = self.export_plain_tx(name, enckey, &hex::encode(tx_info.tx.id()));
```
Comment (calvinaco) [inline client-core/src/wallet/default_wallet_client.rs:1070]: `export_plain_tx` is slightly overkill for checking transaction existence. Maybe a method to get only/check existence of the transaction from storage?

Comment (leejw51crypto) [reply client-core/src/wallet/default_wallet_client.rs:1070]: ok i will modifiy it

**Comment by leejw51crypto** [comment]
added another function to check existence tx in the storage

**Pull request merged**